app/Dockerfile

```diff
@@ -14,11 +14,11 @@ RUN npm install --quiet
 # Bundle app source
 COPY . /usr/src/app
 
-ENV NODE_ENV=production
+ENV STAGING=production
 ENV PORT=4000
 
 RUN ./node_modules/.bin/webpack --config webpack.config.js
 
-EXPOSE 3000
+EXPOSE 4000
```

Comment:
The dockerfile wasn't used for a while. The development make use of webpack dev server. This change was necessary because as it was in the Dockerfiles, app and api where exposing the same ports... Now App is listening on port 4000 and api on port 3000